storage/backend/backend.go

```diff
@@ -0,0 +1,83 @@
+package backend
+
+import (
+	"log"
+	"time"
+
+	"github.com/boltdb/bolt"
+)
+
+type Backend interface {
+	BatchTx() BatchTx
+	ForceCommit()
```

Comment:
Could we remove this function? I see it is only used once in code, and could be replaced by batchTx.Commit.